beaker-vue/src/components/cell/BeakerCell.vue

```diff
@@ -6,6 +6,7 @@
         ref="beakerCellRef"
         :class="{collapsed}"
         :cell-id="cell.id"
+        :style="collapsed ? { '--dynamic-collapse-height': collapseHeight } : {}"
```

Comment:
In relation to truncating the code cell instead of collapsing. Code cells that are longer than a specific height value will truncate to some height that can b equivalent, in most cases, to 10 lines of code. This isn't implemented in the code cell as the collapsed state and toggle are part of BeakerCell.

It does affect the pre-existing notebook interface collapse functionality to also truncate when we press the control to collapse them. This work aimed to modify the "legacy" notebook interface or the common components as least as possible. In this instance it did affect it. My perspective is that the legacy notebook will not be used once this next gen is is tested and becomes the default, but could revisit if needed.